pkg/microservice/aslan/core/service/service/service.go

```diff
@@ -813,6 +813,86 @@ func UpdateServiceTemplate(args *commonservice.ServiceTmplObject) error {
 	return commonrepo.NewServiceColl().Update(updateArgs)
 }
 
+func UpdateServiceHealthCheckStatus(args *commonservice.ServiceTmplObject) error {
+	currentService, err := commonrepo.NewServiceColl().Find(&commonrepo.ServiceFindOption{
+		ProductName: args.ProductName,
+		ServiceName: args.ServiceName,
+		Revision:    args.Revision,
+	})
+	if err != nil {
+		log.Errorf("Can not find service with option %+v", args)
```

Comment:
`log.Errorf("Can not find service with option %+v. Error: %s", args, err)`?

Reply:
changed